force-app/main/default/layouts/Contact-Program Management.layout-meta.xml

```diff
@@ -310,9 +310,12 @@
         <fields>NAME</fields>
         <fields>DeliveryDate__c</fields>
         <fields>Quantity__c</fields>
+        <fields>AttendanceStatus__c</fields>
```

Comment:
We probably want this before quantity or after unit of measure, since quantity is kind of useless without UoM. Given that only the first four show up on the record home, let's say before Quantity.

Reply:
Checked with Claire on separating quantity from uOM , she thinks even though it is not ideal it’s better than not displaying attendance on the page, so she is ok with how it is displayed and attendance status as the last field that’s displayed.